Tabloid/Repositories/ITagRepository.cs

```diff
@@ -6,5 +6,8 @@ namespace Tabloid.Repositories
     public interface  ITagRepository
     {
         List<Tag> GetAllTags();
+
+        void Add(Tag tag);
+        Tag getTagById(int id);
```

Comment:
This method should be PascalCase, not camelCase named

Reply:
@joshdbarton changed from camelCase to PascalCase